relay-server/src/extractors/signed_json.rs

```diff
@@ -91,7 +95,7 @@ impl<T: DeserializeOwned + 'static> FromRequest<ServiceState> for SignedJson<T>
                     .relay
                     .ok_or_else(|| Error::from(SignatureError::UnknownRelay))
             })
-            .join(req.body().map_err(Error::from))
+            .join(RequestBody::new(req, MAX_JSON_SIZE).map_err(Error::from))
```

Comment:
This is an important change. We used the raw Actix `MessageBody`, which does not finish consuming if there is an error while streaming. By using `RequestBody`, we now guarantee to finish consuming. The maximum size of 256kB comes from Actix' defaults.